<?xml version="1.0"?>
<launch>
  <!-- Define arguments for joy_type and cam topics -->
  <arg name="joy_type" default="deck" />  <!-- deck or ds4 -->
  <arg name="cam1_topic" default="/front/usb_cam/image_raw" />
  <arg name="cam2_topic" default="/left/usb_cam/image_raw" />
  <arg name="cam3_topic" default="/right/usb_cam/image_raw" />


  <node pkg="baemin_operator" type="baemin_operator" name="baemin_operator" output="screen">
    <param name="cam1_topic" type="string" value="$(arg cam1_topic)" />
    <param name="cam2_topic" type="string" value="$(arg cam2_topic)" />
    <param name="cam3_topic" type="string" value="$(arg cam3_topic)" />
  </node>
  
  <!-- <include file="$(find base_description)/launch/kuls.launch" />
  <include file="$(find mobile_base_udp)/launch/operator_udp.launch" />
  <include file="$(find mobile_base_joy)/launch/$(arg joy_type).launch"/>
  <include file="$(find audio_play)/launch/play.launch"/> -->

</launch>